<launch>

    <!-- start SAETTA system using platform simulator -->
    <!--<node name="saetta_simulator" pkg="saetta_energy" type="simulator.py" output="screen"/>-->
    <!--<node name="action_tracker" pkg="saetta_energy" type="tracker.py" output="screen"/>-->

    <node name="energy_monitor" pkg="saetta_energy" type="node_energy_monitor.py" output="screen">
        <param name="battery_capacity" value="1800" type="double"/>
        <param name="ts_publish" value="2.0" type="double"/>
        <param name="ts_update" value="0.5" type="double"/>
    </node>

</launch>